metadata.json

```diff
@@ -31,7 +31,7 @@
     }
   ],
   "requirements": [
-    {"name":"puppet","version_requirement":">= 3.0.0 < 5.0.0" }
+    {"name":"puppet","version_requirement":">= 4.0.0 < 5.0.0" }
   ],
   "dependencies": [
     {"name":"puppetlabs/stdlib","version_requirement":">= 4.1.0 < 5.0.0"}
```

Comment:
We should probably bump stdlib to the latest at this point, which we will want for the data types.

https://github.com/puppetlabs/puppetlabs-stdlib#data-types

This will mean we also need to modify the .fixtures.yml